git/go.mod

```diff
@@ -2,121 +2,22 @@ module github.com/fluxcd/pkg/git
 
 go 1.17
 
-replace (
-	github.com/fluxcd/pkg/gittestserver => ../gittestserver
-	github.com/fluxcd/pkg/gitutil => ../gitutil
-	github.com/fluxcd/pkg/http/transport => ../http/transport
-	github.com/fluxcd/pkg/runtime => ../runtime
-	github.com/fluxcd/pkg/ssh => ../ssh
-	github.com/fluxcd/pkg/version => ../version
-)
-
-// Fix CVE-2022-1996 (for v2, Go Modules incompatible)
-replace github.com/emicklei/go-restful => github.com/emicklei/go-restful v2.16.0+incompatible
-
-// A temporary fork of git2go was created to enable use
-// of libgit2 without thread support to fix:
-// fluxcd/image-automation-controller/#339.
-//
-// This can be removed once libgit2/git2go#918 is merged.
-//
-// The fork automatically releases new patches based on upstream:
-// https://github.com/pjbgf/git2go/commit/d72e39cdc20f7fe014ba73072b01ba7b569e9253
-replace github.com/libgit2/git2go/v33 => github.com/pjbgf/git2go/v33 v33.0.9-nothread-check
-
 require (
-	github.com/Masterminds/semver/v3 v3.1.1
 	// github.com/ProtonMail/go-crypto is a fork of golang.org/x/crypto
 	// maintained by the ProtonMail team to continue to support the openpgp
 	// module, after the Go team decided to no longer maintain it.
 	// When in doubt (and not using openpgp), use /x/crypto.
 	github.com/ProtonMail/go-crypto v0.0.0-20220714114130-e85cedf506cd
 	github.com/cyphar/filepath-securejoin v0.2.3
-	github.com/fluxcd/gitkit v0.6.0
-	github.com/fluxcd/pkg/gittestserver v0.5.4
-	github.com/fluxcd/pkg/gitutil v0.1.0
-	github.com/fluxcd/pkg/http/transport v0.0.1
-	github.com/fluxcd/pkg/runtime v0.16.2
-	github.com/fluxcd/pkg/ssh v0.5.0
-	github.com/fluxcd/pkg/version v0.1.0
-	github.com/go-git/go-billy/v5 v5.3.1
-	github.com/go-git/go-git/v5 v5.4.2
-	github.com/go-logr/logr v1.2.3
-	github.com/libgit2/git2go/v33 v33.0.9
 	github.com/onsi/gomega v1.19.0
-	golang.org/x/crypto v0.0.0-20220427172511-eb4f295cb31f
-	golang.org/x/net v0.0.0-20220225172249-27dd8689420f
-	sigs.k8s.io/controller-runtime v0.11.2
+	golang.org/x/crypto v0.0.0-20220427172511-eb4f295cb31f // indirect
+	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
```

Comment:
Something looks weird in the mod file change.
These indirect dependencies are usually added in a separate require section below.
And then previously we didn't had `kr/text` and `gopkg.in/check.v1` dependencies. I can't even find any other dependency here that imports them. Tried deleting them and running tidy, looks like they are not required by anyone.